### ISSUE TITLE
Deprecate sage.homology.graded_resolution.GradedFreeResolution

### ISSUE THREAD

**Issue by kwankyu** [sagemath/sage#34873]
<div id="comment:0"></div>

which we advertised in

https://trac.sagemath.org/wiki/ReleaseTours/sage-9.7#Algebra

CC:  @tscrim

Component: **algebra**

Author: **Kwankyu Lee**

Branch/Commit: **[u/klee/deprecate_sage_homology_graded_resolution_gradedfreeresolution](https://github.com/sagemath/sagetrac-mirror/tree/u/klee/deprecate_sage_homology_graded_resolution_gradedfreeresolution) @ [`349f950`](https://github.com/sagemath/sagetrac-mirror/commit/349f950b0881d062e70454a181ff986505551cf1)**

Reviewer: **Travis Scrimshaw**

_Issue created by migration from https://trac.sagemath.org/ticket/34873_

**Comment by kwankyu** [comment]
Branch: **[u/klee/deprecate_sage_homology_graded_resolution_gradedfreeresolution](https://github.com/sagemath/sagetrac-mirror/tree/u/klee/deprecate_sage_homology_graded_resolution_gradedfreeresolution)**

**Comment by kwankyu** [comment]
Author: **Kwankyu Lee**

**Comment by kwankyu** [comment]
Changed branch from **[u/klee/deprecate_sage_homology_graded_resolution_gradedfreeresolution](https://github.com/sagemath/sagetrac-mirror/tree/u/klee/deprecate_sage_homology_graded_resolution_gradedfreeresolution)** to none

**Comment by kwankyu** [comment]
Branch: **[u/klee/deprecate_sage_homology_graded_resolution_gradedfreeresolution](https://github.com/sagemath/sagetrac-mirror/tree/u/klee/deprecate_sage_homology_graded_resolution_gradedfreeresolution)**

**Comment by kwankyu** [comment]
<div id="comment:3"></div>

New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/349f950b0881d062e70454a181ff986505551cf1"><code>349f950</code></a></td><td><code>Deprecate GradedFreeResolution</code></td></tr></table>

**Comment by kwankyu** [comment]
Commit: **[`349f950`](https://github.com/sagemath/sagetrac-mirror/commit/349f950b0881d062e70454a181ff986505551cf1)**

**Comment by kwankyu** [comment]
Description changed:
``````diff
--- 
+++ 
@@ -1 +1,3 @@
+which we advertised in
 
+https://trac.sagemath.org/wiki/ReleaseTours/sage-9.7#Algebra
``````

**Comment by tscrim** [comment]
<div id="comment:7" align="right">comment:7</div>

LGTM.

**Comment by tscrim** [comment]
Reviewer: **Travis Scrimshaw**

**Comment by kwankyu** [comment]
<div id="comment:8" align="right">comment:8</div>

Thanks!

**Comment by kwankyu** [comment]
Merged to Sage 9.8.rc1
